index.js

```diff
@@ -4,13 +4,13 @@ const electron = require('electron');
 
 let menu = null;
 
-function inpectMenuTemplate(pos) {
+function inpectMenuTemplate(elm, pos) {
   return {
     label: 'Inspect element',
     click: () => {
-      electron.remote
-        .getCurrentWindow()
-        .inspectElement(pos.x, pos.y);
+        const win = elm.tagName.toLowerCase() === 'webview' ?
+            elm : electron.remote.getCurrentWindow()
+        win.inspectElement(pos.x, pos.y);
```

Comment:
Ah you mean just this ?

```
    elm..inspectElement(pos.x, pos.y);
```

We can skip the `electron.remote.getCurrentWindow()` all together? Okay :) will fix it tomorrow.